cmd/http-check/main.go

```diff
@@ -23,22 +23,26 @@ var (
 func init() {
 	// Check that the URL environment variable is valid.
 	if len(checkURL) == 0 {
-		log.Fatalln("Empty URL. Please update your CHECK_URL environment variable.")
+		log.Errorln("Empty URL. Please update your CHECK_URL environment variable.")
+		os.Exit(0)
```

Comment:
If you `os.Exit(0)`, then kuberhealthy will report that the checker pod did not report in within its timeout.  Let's change these exits to report failure with the same message, then exit.

Reply:
I have refactored these to report to kuberhealthy of the failure before exit